satellite/referrals/service.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package referrals
+
+import (
+	"context"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/rpc"
+	"storj.io/storj/pkg/signing"
+	"storj.io/storj/pkg/storj"
+)
+
+var (
+	// Error is the default error class for referrals package.
+	Error = errs.Class("referrals")
+
+	// ErrReferralsConfigMissing is a error class for reporting missing referrals service configuration
```

Comment:
not sure, but it seems that due to our style guide we should end comment with a dot